ppa/archive/forms.py

```diff
@@ -1,5 +1,42 @@
 from django import forms
+from django.utils.safestring import mark_safe
+
+
+class FacetChoiceField(forms.MultipleChoiceField):
+    '''Add CheckboxSelectMultiple field with facets taken from solr query'''
```

Comment:
Noted that this was used as is, and also noted changes to form methods adapted from the derrida codebase. Also included a quick link in case we needed it back to the repo.